install-dropday.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Name of the Magento container
+CONTAINER_NAME="magento-magento-1"
```

Comment:
This target assumes the project is cloned in a directory called "magento". This project should be able to live in any directory name.

Reply:
This is not the directory name. This is the name of the container which is set prior and will remain same everythime any one setup the code

Reply:
And the name of the container is determine by the folder that is lives in. 

So `magento-magento-1` is based on `{directory}-{service}-{instance}`.

I know this for a fact, because I ran the script from insude dropday-io-magento. And the name became `dropday-io-magento-magento-1`.